app/src/org/commcare/activities/MultimediaInflaterActivity.java

```diff
@@ -162,6 +164,7 @@ private void searchForDefault() {
 
         if (Environment.MEDIA_MOUNTED.equals(Environment.getExternalStorageState()) ||
                 Environment.MEDIA_MOUNTED_READ_ONLY.equals(Environment.getExternalStorageState())) {
+            Logger.log(LogTypes.SOFT_ASSERT, "Access outside scoped storage in searching zip file");

```

Comment:
It seems like this path will definitely get hit if someone is using this functionality. So there is no point in adding soft assert here and we will likely need to disable this functionality in Android 11 if there is no other alternate provided in Android 11 for file searching.